desktop/apps/auction2/client/index.js

```diff
@@ -11,7 +11,7 @@ import { setupCommercialFilter } from './commercial_filter'
 import { data as sd } from 'sharify'
 import _ from 'underscore'
 
-const auction = new Auction(_.pick(sd.AUCTION, 'start_at', 'live_start_at', 'end_at', 'associated_sale'))
```

Comment:
Given Backbone models are designed around a single full JSON response, I'm not surprised this caused issues. This is also something to be mindful of if one is wrapping Metaphysics responses in Backbone.